xml/System.ServiceModel.Syndication/SyndicationFeed.xml

```diff
@@ -1924,9 +1929,30 @@
         <ReturnType>System.Collections.ObjectModel.Collection&lt;System.String&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a collection of strings indicating the set of values in the 'skipDays' element for the feed.</summary>
+        <value>A collection of strings indicating the days when aggregators should skip updating the feed.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ When serialized to RSS 2.0, the <xref:System.ServiceModel.Syndication.SyndicationFeed.SkipDays%2A> property is written to a `<skipDays>` element
+ with subelements for each day in the collection.
+
+     
+## Examples  
+ The following XML shows how <xref:System.ServiceModel.Syndication.SyndicationFeed.SkipHours%2A> is serialized to RSS 2.0.  
+  
+```  
```

Comment:
We should double check the preview of these remark nested `<item>` elements, they should look like actual xml code, and not get unexpectedly misinterpreted like something else.

@mairaw do you know if the remarks markdown code would accept specifying the xml programming language like this?:

```suggestion
```xml 
```